arlo_server/superadmin.py

```diff
@@ -46,3 +50,238 @@ def superadmin_jurisdictionadmin_login():
     user_email = request.form["email"]
     set_loggedin_user(UserType.JURISDICTION_ADMIN, user_email)
     return redirect("/")
+
+
+def new_id():
+    return str(uuid.uuid4())
+
+
+def clone_user_with_prefix(user, prefix):
+    prefix = prefix.lower()
+    if user.email.startswith(prefix):
+        return user
+
+    new_email = f"{prefix}{user.email}"
+    new_user = User.query.filter_by(email=new_email).one_or_none()
+
+    if not new_user:
+        new_user = User(id=new_id(), email=new_email)
+        db.session.add(new_user)
+
+    return new_user
+
+
+@app.route("/superadmin/election-clone", methods=["POST"])
+@with_superadmin_access
+def superadmin_election_clone():
```

Comment:
i am worried that we will fail to keep this up to date when the data model changes

Reply:
maybe a simpler approach would be to add `election_id` to every table

Reply:
for sure this will fail when data model changes. It needs at least a test that tries to do this and checks a few things... maybe the serialization of the election at least?

Could add `election_id` to every table, but that will make for some composite keys all over the place unless you want a potentially inconsistent data model. I don't mind composite keys but I hear you like them less ;)